src/core/thread/fiber.d

```diff
@@ -518,7 +518,10 @@ private
  *
  * Authors: Based on a design by Mikola Lysenko.
  */
-class Fiber
+
+import core.thread.context : StackContext, StackContextExecutor;
```

Comment:
You have multiple import of the same module just collapse them into one, and the public import for `Rethrow` just do `alias Rethrow = .Rethow;` (or `alias Rethrow = core.thread.context.Rethrow;` if that doesn't work)

Reply:
Yes, the imports are a little bad at the moment, along with the versioning. The last one is good thought, thanks.